CMakeLists.txt

```diff
@@ -64,12 +67,50 @@ if (NOT ${qBittorrent_VERSION_TWEAK} EQUAL 0)
 endif()
 set(QBT_PROJECT_VERSION "${QBT_PROJECT_VERSION}${QBT_VER_STATUS}")
 
+set(HAS_COMMIT_HASH FALSE)
+if (DEFINE_COMMIT_HASH)
+    execute_process(COMMAND git rev-parse --short=10 HEAD
```

Comment:
Why not use the default hash length? Are there specific reasons to use `10`?
ps. I would prefer the default length.